client/src/templates/Challenges/components/tool-panel.css

```
@@ -11,11 +11,43 @@
 .tool-panel-group-mobile {
   display: flex;
   flex-direction: row-reverse;
-  padding: 0 2px;
 }
 
-.tool-panel-group-mobile .btn-block + .btn-block {
+.tool-panel-group-mobile > .btn-block,
+.tool-panel-group-mobile > .dropdown > .btn-block {
   margin: 0 2px 0 0;
-  padding-left: 8px;
-  padding-right: 8px;
+  padding: 5px 0px;
+}
+
+.tool-panel-group .btn-group {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.tool-panel-group-mobile .btn-group {
+  display: block;
+  width: 100%;
+  margin: 0 2px 0 2px;
+}
+
+.tool-panel-group-mobile .btn {
+  margin-bottom: 0 !important;
+}
+
+.tool-panel-group .dropdown-menu {
+  top: auto;
+  bottom: 100%;
+  width: calc(100% + 2px);
+  text-align: center;
+  min-width: 240px;
+  left: -1px;
+}
+
+.tool-panel-group .dropdown-menu a {
+  padding: 0.5rem;
+}
+
```

```suggestion

/* Prevent 'sticky hover' effects for mobile devices */
@media (hover: none) {
  .dropdown-toggle.btn-primary:hover {
    color: var(--secondary-color);
    background-color: var(--quaternary-background);
  }
}

```
This should get rid of the hover effect on mobile.

---

That works great, thanks!